File: Model/src/main/java/DTO/ParticipantDTO.java
package DTO;

import java.io.Serializable;

public class ParticipantDTO implements Serializable {
    private String name;
    private int age;
    private int eventCount;

    public ParticipantDTO(String name, int age, int eventCount) {
        this.name = name;
        this.age = age;
        this.eventCount = eventCount;
    }

    public String getName() {
        return name;
    }

    public int getAge() {
        return age;
    }

    public int getEventCount() {
        return eventCount;
    }
}

File: Model/DTO/ParticipantDTO.cs
using System;

namespace mpp_proiect_csharp_DianaGliga11.Model.DTO;

public class ParticipantDTO : Entity<long>
{
    public String name { get; set; }
    public int age {get; set;}
    public int eventCount {get; set;}

    public ParticipantDTO(string name, int age, int eventCount)
    {
        this.name = name;
        this.age = age;
        this.eventCount = eventCount;
    }
}